pkg/proxy/ipvs/proxier.go

```diff
@@ -1310,31 +1319,31 @@ func (proxier *Proxier) syncProxyRules() {
 				continue
 			}
 
-			var lps []netutils.LocalPort
-			for _, address := range nodeAddresses {
-				lp := netutils.LocalPort{
-					Description: "nodePort for " + svcPortNameString,
-					IP:          address,
-					IPFamily:    localPortIPFamily,
-					Port:        svcInfo.NodePort(),
-					Protocol:    netutils.Protocol(svcInfo.Protocol()),
-				}
-				if utilproxy.IsZeroCIDR(address) {
-					// Empty IP address means all
-					lp.IP = ""
-					lps = append(lps, lp)
-					// If we encounter a zero CIDR, then there is no point in processing the rest of the addresses.
-					break
-				}
-				lps = append(lps, lp)
-			}
+			//var lps []netutils.LocalPort
```

Comment:
can you completely remove the lines? not commenting them?

Reply:
okay